gen3/bin/kube-setup-secrets.sh

```diff
@@ -101,19 +101,42 @@ if [[ -f "${WORKSPACE}/${vpc_name}_output/creds.json" ]]; then # update fence se
     g3kubectl create configmap fence --from-file=apis_configs/user.yaml
   fi
 
-  if ! g3kubectl get secrets/fence-secret > /dev/null 2>&1; then
-    g3kubectl create secret generic fence-secret "--from-file=local_settings.py=${GEN3_HOME}/apis_configs/fence_settings.py" "--from-file=${GEN3_HOME}/apis_configs/config_helper.py"
-  fi
+  if ! g3kubectl get secrets/fence-config > /dev/null 2>&1; then
```

Comment:
Would you put some comments here explaining the relationship between `fence_config` and `fence_user_config` ?
It looks like `fence_user_config` is an intermediate file that gets generated on the way to generating `fence_config`?